src/views/AccountDetails/AccountDetails.js

```diff
@@ -45,7 +45,7 @@ import {
 } from '../../constants';
 
 import css from './AccountDetails.css';
-import { localizeCurrencyAmount } from '../../components/util/localizeCurrencyAmount';
+import localizeCurrencyAmount from '../../components/util/localizeCurrencyAmount';
```

Comment:
after updating `src/hooks/index.js` file on the lines of other hooks exports (as suggested below in one of the comments), adjust this import